src/kixi/datastore/metadatastore/inmemory.clj

```diff
@@ -33,9 +65,11 @@
       (if-not data
         (let [new-data (atom {})]
           (info "Starting InMemory Metadata Store")
-          (attach-sink-processor communications
-                                 #(s/valid? ::ms/filemetadata %)
-                                 (update-metadata-processor new-data))
+          (c/attach-event-handler! communications
+                                   :metadatastore
```

Comment:
I may have misunderstood, but didn't you say that this should be namespaced as well?